README.md

```diff
@@ -1,26 +1,25 @@
-# Space Travelers Hub
+# Travel-Guide
 
 <img alt="Version" src="https://img.shields.io/badge/version-1.0.0-blue.svg?cacheSeconds=2592000" />
 
-> This website is for a company that provides commercial and scientific space travel services. The website allow users to book rockets and join selected space missions.
-> 
-It is created with [Create-React-App](https://github.com/facebook/create-react-app)
+> This website is meant to be a travel guide with informations about the european countries. The data is fetched from an external API.
+> It is created with [Create-React-App](https://github.com/facebook/create-react-app)
 
 ---
 
 ## Screenshot
 
-![missionspage](https://user-images.githubusercontent.com/12524453/168415237-3fdd2745-9e46-41cb-befa-b061b5f9b02a.png)
+![All Countries](./src/Assets/countries.jpg) | ![All Countries](./src/Assets/details.jpg)
 
 ## Live version
 
-For the live version of this project visit the following [Netlify](https:// .app/) and [Heroku](https:// .herokuapp.com/) links.
+For the live version of this project visit the following [Netlify](https://harmonious-elf-f99081.netlify.app) 
 
 ## Get started
 
 ```bash
-   git clone git@github.com:Mesi21/Space-Travelers.git
-   cd into the folder by typing: cd Space-Travelers
+   git clone git@github.com:Mesi21/Travel-Guide.git
+   cd into the folder by typing: cd Travel-Guide
```

Comment:
- [ ] Your live demo does not seem to work as expected. I would recommend you fix it.

  ![mesi-live-demo](https://user-images.githubusercontent.com/36542986/170501225-8b5315eb-033e-44bd-9283-4dae2f41f18f.jpg)